### PR TITLE
Spelling: account name, or any e-mail address, -!

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: weblate/trans/forms.py [hunk 1]
```diff
@@ -1043,8 +1043,7 @@ class UserManageForm(forms.Form):
     user = UserField(
         label=_('User to add'),
         help_text=_(
-            'Please provide username or e-mail. '
-            'User needs to already have an active account in Weblate.'
+            'Please type in an account name, or any e-mail address. '
```
Comment (nijel) [inline weblate/trans/forms.py]: This removes information that this is only for existing users.

Comment (comradekingu) [inline weblate/trans/forms.py]:
```suggestion
            'Please type in a Weblate account name, or any e-mail address. '
```

File: weblate/trans/forms.py [hunk 2]
```diff
@@ -1043,8 +1043,7 @@ class UserManageForm(forms.Form):
     user = UserField(
         label=_('User to add'),
         help_text=_(
-            'Please provide username or e-mail. '
-            'User needs to already have an active account in Weblate.'
+            'Please type in a Weblate account name, or any e-mail address. '
```
Comment (nijel) [inline weblate/trans/forms.py]:
```suggestion
            'Please type in an username or e-mail address of existing Weblate account. '
```

How about this? It should be clear that this is only for existing accounts.

Comment (comradekingu) [inline weblate/trans/forms.py]:
```suggestion
            'Please type in an existing Weblate account name or e-mail address. '
```

**Comment by nijel** [comment]
Merged, thanks for your contribution!

I've removed trailing space in 6723b2f0935f11463036c3ee06495e8197364514.

**Pull request merged**